main.tf

```diff
@@ -86,3 +91,299 @@ resource "databricks_notebook" "notebook" {
   overwrite = false
   format    = "JUPYTER"
 }
+
+resource "azurerm_api_management_api" "notebook_api" {
```

Comment:
As the notebook is conditional - does this API not need to be too?

Reply:
Good spot. Made API conditional on notebook_path variable and tested manually. 🙂